uaclient/cli.py

```diff
@@ -246,6 +246,7 @@ def action_fix(args, cfg, **kwargs):
             "Invalid issue format: {}".format(args.security_issue)
         )
     security.fix_security_issue_id(cfg, args.security_issue)
+    return 0
```

Comment:
We won't get here if there are any exceptions of any type because we raise X in that case... when that happens [cli.py:main_error_handler](https://github.com/canonical/ubuntu-advantage-client/blob/master/uaclient/cli.py#L967) takes over which inevitably calls sys.exit(1)